web/.husky/pre-commit

```diff
@@ -1,4 +1,35 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 . "$(dirname -- "$0")/_/husky.sh"
```

Comment:
Maybe this should be moved to the block of `web_modified`?

Reply:
Seems it's part of Husky required for Windows. I would prefer not to change it for this time.
https://typicode.github.io/husky/troubleshoot.html#yarn-on-windows